mitodl/ocw-hugo-themes
# updated directory for testimonials to stories

#### Pre-Flight checklist

#### What are the relevant tickets?
Part of #614 

#### What's this PR do?
Renames the testimonials directory to stories and updates home_testimonials.html to get stories

#### How should this be manually tested?
- Verify `View All OCW Stories` opens index page for stories
- Verify testimonials work as it is.

#### Any background context you want to provide?
This PR is needs to be merged with the following PRs
- https://github.com/mitodl/ocw-hugo-projects/pull/154
- https://github.com/mitodl/ocw-studio/pull/1250
